content/docs/rfcs/32/README.md

```diff
@@ -0,0 +1,365 @@
+---
+slug: 32
+title: 32/RLN-SPEC
+name: RLN specification
+status: raw
+editor: Blagoj Dimovski <blagoj.dimovski@yandex.com>
+contributors:
+---
+
+# Abstract
+
+The following document covers the specification of the RLN construct as well as the auxiliary libraries necessary for interaction with it.
+
+# Motivation
+
+RLN (Rate limiting nullfier) is a construct based on zero-knowledge proofs that enables spam prevention mechanism for decentralized, anonymous environments. In anonymous environments, the identity of the entities is unknown, and thus it is very hard to prevent spam attacks. Many applications would benefit from anonimity, but potential for spam could worsen their UX by large extent. RLN allows for anonymous signaling, while enabling spam prevention on protocol level.
```

Comment:
I suggest adding the definition of spam.

Reply:
Also, I think it would be good to introduce RLN as a  rate-limiting tool and mention spam protection as one of its applications e.g., it can also be used in e-voting.

Reply:
I agree, I would add those definitions and change the section accordingly.